_data/sites.json

```
@@ -8496,5 +8496,16 @@
         "domains": [
             "zynga.com"
         ]
+    },
+    
+    {
+        "name": "/e/",
+        "url": "https://doc.e.foundation/create-an-ecloud-account#how-can-i-delete-my-e-account-eemail",
+        "difficulty": "hard",
+        "notes": "Send an email to contact@e.email from the /e/ID (<yourmail>@e.email) you want to delete. Optionally, you can describe the reason for the exclusion.",
```

Because the `email` field is provided, a "Send email »" link will be present under the notes, so there is no need to specify the contact email.

HTML entities in the notes field are displayed as-is, so `<>` must be replaced by their escaped entities to avoid telling the browser to try to display a `<yourmail>` element.
```suggestion
        "notes": "Send an email from the /e/ID (&lt;yourmail&gt;@e.email) you want to delete. Optionally, you can describe the reason for the exclusion.",
```